forloop_modules/queries/db_model_templates.py

```diff
@@ -201,9 +201,14 @@ class APIFile(BaseModel):
 
 class APIVariableToBeAssignedToFile(BaseModel):
     variable_name: str = ""
+    node_uid: str = ""
     project_uid: str = "0"
     pipeline_uid: str = "0"
+    
 
+class APILoadScriptViaNode(BaseModel):
+    node_uid: str = ""
```

Comment:
node_uid = "0" is maybe better? -> then, if 0 is passed anywhere - it means you didnt specify it where you should do that :)